rewrite-java/src/main/java/org/openrewrite/java/cleanup/SimplifyConstantIfBranchExecution.java

```diff
@@ -46,6 +47,8 @@ public String getDescription() {
     }
 
     private static class SimplifyConstantIfBranchExecutionVisitor extends JavaVisitor<ExecutionContext> {
+        private static final UnnecessaryParenthesesVisitor UNNECESSARY_PARENTHESES_VISITOR =
```

Comment:
[TreeVisitor](https://github.com/openrewrite/rewrite/blob/6c391098cb5363bc4bdf3316940013e0dfbd8e28/rewrite-core/src/main/java/org/openrewrite/TreeVisitor.java#L50) is not thread-safe, remove this variable declaration and use a new instance for each operation